test/services/bill-runs/two-part-tariff/fetch-bill-run-licences.service.test.js

```diff
@@ -24,36 +24,41 @@ describe('Fetch Bill Run Licences service', () => {
   describe('when there is a valid bill run', () => {
     let billRun
     let region
-    let licenceHolder
-    let licenceStatus
+    let filterLicenceHolder
```

Comment:
Maybe move these two 'let filterLicenceHolder' and 'let filterLicenceStatus' to be with 'filterIssues' on line 36 just for consistency?